python/tvm/relay/frontend/tflite.py

```diff
@@ -320,6 +321,45 @@ def dequantize(self, expr, tensor):
                                          input_zero_point=tensor.qnn_params['zero_point'])
         return dequantized
 
+
+    def convert_qnn_fused_activation_function(self, expr, fused_activation_fn,
+                                              scale, zero_point, dtype):
+        """Convert TFLite fused activation function. The expr is an input quantized tensor with
+        scale and zero point """
```

Comment:
I think this could be a separate PR as it's not specific to object detection.

Reply:
This is needed for quantized object detection model

Reply:
It's more from a position of having the history clearly reflect when features were added. Supporting fused qnn functions is needed for this model, but it's also necessary for a wide range of other models so I'd view this as a feature not specific to object detection.